Source/Core/PolylineGeometry.js

```diff
@@ -361,6 +362,12 @@ define([
                 ellipsoid: ellipsoid,
                 height: heights
             });
+        } else {
+            // This subdivides a line at a much greater granularity. This may be a workaround for an actual bug
+            // where a polyline blinks in and out during camera movement when log depth is enabled and a segment of
+            // the polyline intersects the near plane.
+            minDistance = CesiumMath.chordLength(granularity * 10.0, ellipsoid.maximumRadius);
```

Comment:
I don't have 100% context here, but shouldn't this ignore `granularity` and just pick a min distance to subdivide, which could be as simple as `ellipsoid.maximumRadius`?  Previous behavior was that `granularity` was ignored unless the polyline follows the surface - why would that change now - when would a user want to explicitly provide `granularity` for non-surface lines - it doesn't even make sense as a measurement for arbitrary lines, right?